lib/peep.rb

```diff
@@ -0,0 +1,25 @@
+require 'pg'
+
+class Peep
+
+    def self.all
+        if ENV['ENVIRONMENT'] == 'test'
+            connection = PG.connect(dbname: 'chitter_test')
+        else 
+            connection = PG.connect(dbname: 'chitter')
+        end
+
+        result = connection.exec('SELECT * FROM peeps;')
+        result.map{ |peep| peep["message"] }
+    end
+
+    def self.post(message)
+        if ENV['ENVIRONMENT'] == 'test'
+            connection = PG.connect(dbname: 'chitter_test')
+        else 
+            connection = PG.connect(dbname: 'chitter')
+        end
```

Comment:
This piece of code appears twice in this file. How might you remove this duplication?